lib/elastic_apm/util.rb

```diff
@@ -15,5 +15,13 @@ def self.git_sha
       sha = `git rev-parse --verify HEAD 2>&1`.chomp
       $? && $?.success? ? sha : nil # rubocop:disable Style/SpecialGlobalVars
     end
```

Comment:
I suggest to add the `.utc` inside the `def self.micros` method, to ensure all timestamps are used in utc.